op-node/service.go

```diff
@@ -191,8 +191,16 @@ func NewDriverConfig(ctx *cli.Context) *driver.Config {
 }
 
 func NewRollupConfig(log log.Logger, ctx *cli.Context) (*rollup.Config, error) {
-	network := ctx.String(flags.Network.Name)
-	rollupConfigPath := ctx.String(flags.RollupConfig.Name)
+	var network, rollupConfigPath string
```

Comment:
Remove - we shouldn't be changing flag names.

Reply:
done